.github/workflows/publish-conan-branch-package.yml

```diff
@@ -38,15 +38,20 @@ jobs:
           sudo apt-get update
           sudo apt-get install -y jq python3 python3-pip
 
-      - name: Add repos for for gcc-13 and clang-16
+      - name: Add repos for for gcc-13 and clang-16,17
         run: |
           # gcc-13
           sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
 
           # clang-16
           source /etc/os-release
           echo "deb http://apt.llvm.org/${UBUNTU_CODENAME}/ llvm-toolchain-${UBUNTU_CODENAME}-16 main" | sudo tee /etc/apt/sources.list.d/llvm-16.list
-          curl https://apt.llvm.org/llvm-snapshot.gpg.key | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/llvm-16.gpg > /dev/null
+          
+          # clang-17
+          echo "deb http://apt.llvm.org/${UBUNTU_CODENAME}/ llvm-toolchain-${UBUNTU_CODENAME}-17 main" | sudo tee /etc/apt/sources.list.d/llvm-17.list
+          
+          # llvm key
+          curl https://apt.llvm.org/llvm-snapshot.gpg.key | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/llvm.gpg > /dev/null
```

Comment:
I believe this should work but please test it with https://github.com/dice-group/cpp-conan-release-reusable-workflow-test

Reply:
ah, I see. it's already tested with: https://github.com/rdf4cpp/rdf4cpp/pull/225